website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_emr_cluster"
+sidebar_current: "docs-alicloud-resource-emr-cluster"
+description: |-
+  Provides a EMR Cluster resource.
+---
+
+# alicloud\_emr\_cluster
+Provides a EMR Cluster resource. With this you can create, read, and release  EMR Cluster.
+
```

Comment:
please add a note Available in 1.57.0+

Reply:
it should be -> **NOTE:** Available in 1.57.0+ and on a new line

Reply:
At present, the version should be 1.57.0+